### DISCUSSION TITLE
Kaleido M10 Duo Bluetooth connection error (Artisan COM port error)

### DISCUSSION THREAD

**Discussion by MarcelPr** [artisan-roaster-scope/artisan#1199] [Q&A]
I got my M10 DUO a month ago, and in the beginning everything worked fine.
However, lately I had problems with establishing the Bluetooth connection.
Artisan tries and reports the COM port error.

Was able to connect with the tablet disconnected from the roaster. The Bluetooth 
device in this case had a different name Roaster_SRS. But everything worked just fine without
the tablet.

Now, I can conjecture that in order for Artisan to successfully connect with M10 duo,
both tablet and Artisan have to have the same temperature scale, either F or C.  If they
are different, Artisan says "Com port error".

I think it can be fixed.

**Setup (please complete the following information):**
 - Artisan Version 2.8.3.beta
 - Artisan Build 302963
 - Platform (Mac/Windows/Linux + OS version) Windows 11
 - Connected devices or roasting machine Kaleido M10

**Comment by Terracotta-62** [comment]
Download RC version to try, Bluetooth configuration refer to the user manual to reconfigure.
